cdap-ui/app/cdap/components/Administration/AdminConfigTabContent/SystemProfilesAccordion/index.js

```diff
@@ -111,3 +106,23 @@ export default class SystemProfilesAccordion extends Component {
     );
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
+  return {
+    // We have to get profiles count from ownProps to show on the accordion label, instead
+    // of getting from store state, if ProfilesStore is not initialized. This store is only
+    // initalized when ProfilesListView component is mounted, which only happens when the
+    // accordion is expanded
+    profilesCount: state.profiles.length || ownProps.profiles.length
+  };
+};
+
+const ConnectedSystemProfilesAccordion = connect(mapStateToProps)(SystemProfilesAccordion);
+
+export default function SystemProfilesAccordionFn({...props}) {
```

Comment:
i don't think you need the object extend in here, simply having `(props)` would work i believe